sw.js

```diff
@@ -21,8 +21,11 @@ self.addEventListener('fetch', (e) => {
         }
         console.log(e.request.url);
         let resp = await caches.match(e.request);
-        if (!resp)
+        if (!resp) {
+            if (e.request.cache === 'only-if-cached')
+                event.request.mode = 'same-origin';
```

Comment:
`e.request.mode`?

Also, could you elaborate on the intention of this change?

Reply:
Oops, you're right.

It was for https://bugs.chromium.org/p/chromium/issues/detail?id=823392 ... but looks like it's an impl bug around DevTools and we can probably just ignore it.

Reply:
Removed this part in the new patch.